api/tests/prescription/campaign/unit/infrastrucutre/repositories/organization-membership-repository_test.js

```diff
@@ -0,0 +1,26 @@
+import { OrganizationMembership } from '../../../../../../src/prescription/campaign/domain/read-models/OrganizationMembership.js';
+import { getByUserIdAndOrganizationId } from '../../../../../../src/prescription/campaign/infrastructure/repositories/organization-membership-repository.js';
+import { expect, sinon } from '../../../../../test-helper.js';
+
+describe('Unit | Repositories | Organization Membership Repository', function () {
+  it('should', async function () {
```

Comment:
**todo**: Should or should not, that is the question

Reply:
**todo** `it should returns the corresponding OrganizationMemberShip Model`